products/BellHybrid/apps/common/src/layouts/HomeScreenLayoutVertical.cpp

```diff
@@ -129,6 +135,17 @@ namespace gui
         return (batteryContext.level < dischargingLevelShowTop) or isBatteryCharging(batteryContext.state);
     }
 
+    bool HomeScreenLayoutVertical::isAlarmTimeVisibilityAllowed()
+    {
+        return false;
+    }
+
+    bool HomeScreenLayoutVertical::isBatteryCharging(const Store::Battery::State state)
+    {
+        using State = Store::Battery::State;
+        return state == State::Charging or state == State::ChargingDone;
```

Comment:
```suggestion
        return (state == State::Charging) || (state == State::ChargingDone);
```

AFAIR in some discussion the team decided to consistently use symbol-like logical operators instead of keyword-like ones in our codebase ;)

Reply:
I agree.